addon/components/materialize-collection-links.js

```diff
@@ -0,0 +1,6 @@
+import materializeCollection from './materialize-collection';
+
+export default materializeCollection.extend({
+  layoutName: 'components/materialize-collection',
+  tagName: 'div',
```

Comment:
I'm specifing `tagName: 'div'` because this component is extended from another one which has `tagName: 'ul'`, so I have to overwrite it